plugin/src/main/java/org/owasp/benchmarkutils/score/BenchmarkScore.java

```diff
@@ -163,140 +118,47 @@ public class BenchmarkScore extends AbstractMojo {
             }
         }
 
-        // Find Default Scoring Config File
-        final String DEFAULTCONFIGFILE = "defaultscoringconfig.yaml";
-        InputStream defYamlFileStream =
-                BenchmarkScore.class.getClassLoader().getResourceAsStream(DEFAULTCONFIGFILE);
-        if (defYamlFileStream == null) {
-            System.out.println(
-                    "ERROR: default YAML scoring configuration file: '"
-                            + DEFAULTCONFIGFILE
-                            + "' not found on classpath!");
-            return false;
+        // TODO: move to html class (once this has been extracted, too)
+        if (!config.includeProjectLink) {
+            PROJECTLINKENTRY = "";
         }
 
-        Yaml yaml = new Yaml();
-        Map<String, Object> yamlConfig = null;
-        if (yamlFileStream != null) {
-            // If specified, load Both Default and Custom YAML file in ONE Stream, so Custom Values
-            // Override Default
-            InputStream fullStream = new SequenceInputStream(defYamlFileStream, yamlFileStream);
-            yamlConfig = yaml.load(fullStream);
-            System.out.print("YAML Scoring config file found and loaded. File used was: ");
-            if (yamlFile != null) { // Not null means the a normal file was used, not a resource
-                System.out.println(yamlFile.getAbsolutePath());
-            } else {
-                try {
-                    System.out.println(
-                            BenchmarkScore.class.getClassLoader().getResource(args[1]).toURI());
-                } catch (URISyntaxException e) {
-                    // This should never happen
-                    System.out.println(
-                            "Couldn't determine URI of existing resource: '"
-                                    + args[1]
-                                    + "' for some reason.");
-                    e.printStackTrace();
-                }
-            }
-        } else { // just load the default file
-            yamlConfig = yaml.load(defYamlFileStream);
-            System.out.println(
-                    "default YAML Scoring config file found and loaded. File used was: ");
-            try {
-                System.out.println(
-                        BenchmarkScore.class
-                                .getClassLoader()
-                                .getResource(DEFAULTCONFIGFILE)
-                                .toURI());
-            } catch (URISyntaxException e) {
-                // This should never happen
-                System.out.println(
-                        "Couldn't determine URI of existing resource: '"
-                                + DEFAULTCONFIGFILE
-                                + "' for some reason.");
-                e.printStackTrace();
-            }
+        if (!config.includePrecision) {
+            // This two values are both included or not included together (currently)
+            PRECISIONKEYENTRY = "";
```

Comment:
Should be 'These two values ...

Reply:
Weird, this is not my comment, but I can fix it

Reply:
Done